scripts/populate_review_step.py

```diff
@@ -30,6 +30,18 @@ def check_rest_data_and_artifacts(self, sample_name, selector):
             )
         return rest_entities, artifacts
 
+    def delivered(self, sample_name):
+        d = {'yes': True, 'no': False}
+        query_args = {'where': {'sample_id': sample_name}}
+        sample = self.get_documents('samples', **query_args)[0]
+        return d.get(sample.get('delivered'))
+
+    def processed(self, sample_name):
+        query_args = {'where': {'sample_id': sample_name}}
+        sample = self.get_documents('samples', **query_args)[0]
+        processing_status = sample.get('aggregated', {}).get('most_recent_proc', {}).get('status')
```

Comment:
We have EGCG-Core 0.8.1 now, so can do something like `util.query_dict(sample, 'aggregated.most_recent_proc.status')`